src/java.base/share/classes/sun/launcher/resources/launcher.properties

```diff
@@ -232,6 +232,21 @@ The following options are macOS specific:\n\
 \    -Xdock:icon=<path to icon file>\n\
 \                      override default icon displayed in dock\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   Usage: java [java options...] <application> [application arguments...]\n\n\
+\Where <application> is one of:\n\
+\  <MainClass>                to execute the main method of a compiled class\n\
+\  -jar <JarFile>.jar         to execute the main class of a JAR archive\n\
+\  -m <module>[/<MainClass>]  to execute the main class of a module\n\
```

Comment:
Did you mean to use capitals here? It will look different to the `--help` output where we consistently use all lowercase.